docs/en/contributing-to-appium/developers-overview.md

```diff
@@ -133,12 +133,28 @@ npm run lint            # run eslint on the code
 npm run watch           # watch directory to re-transpile on code change, lint, and run unit tests
 npm run test            # same as above but don't watch
 npm run e2e-test        # transpile and run end-to-end/functional tests
-_FORCE_LOGS=1 <command> # show module log output during test run
+_FORCE_LOGS=1. <command> # show module log output during test run
 ```
 
 Note that we have a convention for unit test files to end in `-specs.js` and
 for e2e test files to end in `-e2e-specs.js`.
 
+### Updating Appium site docs
+
+The [Appium site](https://appium.io) is built using a site generator that translates the markdown in https://github.com/appium/appium to static HTML
+
+To update the site docs, follow these steps:
+
+1. From Appium repo, run `npm run generate-docs` and make a pull request if there are any newly generated docs.
+1. When the above PR is merged, clone the [appium.io](https://github.com/appium/appium.io) repo (if you haven't done so already)
+1. Check out and pull the latest `gh-pages` branch
+1. Follow the setup guide in the README.md
+1. Generate the docs by running `npm run build:docs`
+1. Test by running `npm run serve` and navigating to https://localhost:4000/
```

Comment:
What does "Test" mean in this context? Are there steps that ought to be gone through to make sure that things are right?

Reply:
I could probably just take this out. I don't think the site needs to be tested, the CI handles that well enough.